web/pandas/about/team.md

```diff
@@ -36,10 +36,19 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+
+## Diversity and Inclusion
+    
+> _pandas_ expressly welcomes and encourages anyone who faces under-representation,discrimination in the technology industry 
+> or anyone willing to increase the diversity of our team,we encourage you to contribute.
+> We have identified the visible diversity gap and obstacles in the open source community and we are strongly against it.
```

Comment:
My personal preference would be to phrase this as a positive message (i.e. avoid using "strongly against it"). I'd use something like "We are aware of the diversity problems in the open source community, and we are proactive in increasing the diversity of our team.". Feel free to use what you think it's better, but I think people react better to messages expressed in a positive way.